examples/oneapi/dpc/BUILD

```diff
@@ -29,13 +29,25 @@ dal_example_suite(
     extra_deps = _TEST_DEPS,
 )
 
+dal_example_suite(
+    name = "kmeans_init",
+    compile_as = [ "dpc++" ],
+    srcs = glob(["source/kmeans_init/*.cpp"]),
+    dal_deps = [
+        "@onedal//cpp/oneapi/dal/algo:kmeans",
+        "@onedal//cpp/oneapi/dal/algo:kmeans_init",
+    ],
+    data = _DATA_DEPS,
+    extra_deps = _TEST_DEPS,
+)
+
 dal_algo_example_suite(
     algos = [
         "basic_statistics",
         "covariance",
         "decision_forest",
         "kmeans",
-        "kmeans_init",
+        "dbscan",
```

Comment:
It's not obvious why kmeans_init was moved and dbscan added here and then removed in list without description

Reply:
kmeans_init was moved to separate dal_example_suite due with conflicts between kmeans and kmeans_init. Dbscan was added due with adding dbscan examples to our bazel examples.